src/containers/pages/FocusInvestigation/active/index.tsx

```diff
@@ -436,26 +432,29 @@ const mapStateToProps = (state: Partial<Store>, ownProps: any): DispatchedStateP
     ownProps.match.params && ownProps.match.params.jurisdiction_parent_id
       ? ownProps.match.params.jurisdiction_parent_id
       : null;
-  const caseTriggeredPlans = getPlansArray(
-    state,
-    InterventionType.FI,
-    [PlanStatus.ACTIVE, PlanStatus.COMPLETE],
-    CASE_TRIGGERED,
-    [],
-    jurisdictionParentId
-  );
-  const routinePlans = getPlansArray(
-    state,
-    InterventionType.FI,
-    [PlanStatus.ACTIVE, PlanStatus.COMPLETE],
-    ROUTINE,
-    [],
-    jurisdictionParentId
-  );
+  const searchString = trimStart(ownProps.location.search, '?');
+  const queryParams = querystring.parse(searchString);
+  const searchedTitle = queryParams.search as string;
+  const caseTriggeredPlans = makePlansArraySelector()(store.getState(), {
+    interventionType: InterventionType.FI,
+    parentJurisdictionId: jurisdictionParentId,
+    reason: CASE_TRIGGERED,
+    statusList: [PlanStatus.ACTIVE, PlanStatus.COMPLETE],
+    title: searchedTitle,
+  });
+  const routinePlans = makePlansArraySelector()(store.getState(), {
```

Comment:
Why are we doing `store.getState()`, do we not already have `state` available here?